ale/drivers/mgs_drivers.py

```diff
@@ -12,6 +12,140 @@
 from ale import util
 
 
+class MgsMocNarrowAngleCameraIsisLabelNaifSpiceDriver(LineScanner, IsisLabel, NaifSpice, RadialDistortion, Driver):
+    """
+    Driver for reading MGS MOC WA ISIS labels.
+    """
+
+    @property
+    def instrument_id(self):
+        """
+        Returns an instrument id for uniquely identifying the instrument, but often
+        also used to be piped into Spice Kernels to acquire IKIDs. Therefore they
+        the same ID the Spice expects in bods2c calls.
+        Expects instrument_id to be defined in the IsisLabel mixin. This should be
+        a string of the form 'MOC-NA'
+
+        Returns
+        -------
+        : str
+          instrument id
+        """
+        id_lookup = {
+        "MOC-NA" : "MGS_MOC_NA"
+        }
+        return id_lookup[super().instrument_id]
+
+
+    @property
+    def sensor_name(self):
+        """
+        Returns
+        -------
+        : String
+          The name of the sensor
+        """
+        return self.instrument_id
+
+    @property
+    def ephemeris_start_time(self):
```

Comment:
I don't think this function needs to be overwritten and should be handled by the IsisLabel and NaifSpice mixins

Reply:
It looks like I actually should have overwritten Spacecraft Clock Start Count here.

Reply:
I guess I am still confused as to why? If you check how ISIS gets the start count it looks for the keyword you are looking for here:
https://github.com/USGS-Astrogeology/ale/blob/e689abf9e5391aa578156a2f957e1499daedb75e/ale/base/label_isis.py#L176

Does the label contain both `SpacecraftClockCount` and `SpacecraftClockStartCount`?

Reply:
Oh, you're right.  I missed that, I thought it only looked for `SpacecraftClockStartCount`.  Removed with commit.